### ISSUE TITLE
Workshop — PWM Issue on RED MSP432P401R LaunchPad 5.6.3

### ISSUE THREAD

**Issue by rei-vilo** [energia/msp432r-core#14]
The same code using PWM for the RGB LED works on the **black** but not on the **red** MSP432P401R LaunchPad.

+ **Black MSP432P401R LaunchPad** with board package 3.8.0

The RGB LEDs display yellow, orange and red colours.

+ **Red MSP432P401R LaunchPad** with board package 5.6.2

The RGB LEDs display none of the colours.

+ Minimal sketch

```c
void setup() {
  // put your setup code here, to run once:
  pinMode(RED_LED, OUTPUT);
  pinMode(GREEN_LED, OUTPUT);
  pinMode(BLUE_LED, OUTPUT);
}

void setRGB(uint8_t red, uint8_t green, uint8_t blue)
{
  analogWrite(RED_LED, red);
  analogWrite(GREEN_LED, green);
  analogWrite(BLUE_LED, blue);  
}

void loop() {
  // put your main code here, to run repeatedly: 
  setRGB(127, 127, 0); // yellow
  delay(1000);
  setRGB(127, 63, 0); // orange
  delay(1000);
  setRGB(127, 0, 0); // red
  delay(1000);  
}
```

**Comment by rei-vilo** [comment]
Just released version 5.6.3 doesn't seem to fix the issue.

**Comment by rei-vilo** [comment]
Solved with release `5.23.1`.
